src/utils.ts

```diff
@@ -10,11 +21,29 @@ export function getPayPalWindowNamespace(
 }
 
 /**
- * Creates a numeric hash based on the string input.
+ * Get a namespace from the window in the browser
+ * this is useful to get the braintree from window
+ * after load Braintree script
+ *
+ * @param namespace the name space to return
+ * @returns the namespace if exists or undefined otherwise
+ */
+export function getBraintreeWindowNamespace(
+    namespace: string = DEFAULT_BRAINTREE_NAMESPACE
+): BraintreeNamespace {
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    return (window as any)[namespace];
+}
+
+/**
+ * Creates a numeric hash based on the string input
+ *
+ * @param str the source string to generate the hash
+ * @returns a hash numeric ode
  */
 export function hashStr(str: string): number {
     let hash = 0;
-    for (let i = 0; i < str.length; i++) {
+    for (let i = 0; i < (str || "").length; i++) {
```

Comment:
Please revert. The string argument is required. 

```suggestion
    for (let i = 0; i < (str).length; i++) {
```